bot/exts/backend/error_handler.py

```diff
@@ -168,27 +173,28 @@ async def try_get_tag(self, ctx: Context) -> None:
         the context to prevent infinite recursion in the case of a CommandNotFound exception.
         """
         tags_get_command = self.bot.get_command("tags get")
+        tags_get_command.can_run = can_run if can_run else self._can_run
         if not tags_get_command:
             log.debug("Not attempting to parse message as a tag as could not find `tags get` command.")
             return
 
-        ctx.invoked_from_error_handler = True
+        interaction.invoked_from_error_handler = True
 
         log_msg = "Cancelling attempt to fall back to a tag due to failed checks."
         try:
-            if not await tags_get_command.can_run(ctx):
+            if not await tags_get_command.can_run(interaction):
```

Comment:
I don't think the try/except should belong here, but in the can_run function instead.
It should be a simple "yes or no" question, and the `CommandError` should be handled inside and return False upon that

Reply:
Just to be clear, I now this not related to your work, but it's only an improvement suggestion

Reply:
Actually, never mind me.
I just realized that this is not part of the lib.
My bad, sorry for that